tests/basic/bug68986.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Bug #68986 (pointer returned by php_stream_fopen_temporary_file not validated in memory.c)
+--INI--
+default_charset=UTF-8
+--FILE--
+<?php
+if (substr(PHP_OS, 0, 3) == 'WIN') {
+	die('skip.. only for unix');
+}
+
+$tempDir = getenv("TMPDIR");
```

Comment:
what if TMPDIR not set? Not every OS in existence has TMPDIR. Then you'd try to write into root directory - probably not a good idea.